docs/.markdownlintignore

```diff
@@ -1 +0,0 @@
-_site/
```

Comment:
I think we want to keep this file but update this directory so that our markdown linter ignores directories that don't contain markdown.